pydm/tests/test_smoke.py

```diff
@@ -0,0 +1,199 @@
+from __future__ import print_function
+import os
+import pkgutil
+import sys
+import types
+import time
+from inspect import isclass, signature, Parameter
```

Comment:
`signature` is Python 3.3+ only 😭

Reply:
`inspect.getargspec` does something similar and works on Python 2 (hilariously, it *doesn't* work in 3.0+), so it may be possible to use that instead.  We do that in application.py somewhere.